plugins/notion/src/notion.ts

```diff
@@ -679,3 +838,51 @@ export function isUnchangedSinceLastSync(lastEditedTime: string, lastSyncedTime:
 
     return lastSynced > lastEdited
 }
+
+interface PaginatedArgs {
+    start_cursor?: string
+}
+
+interface PaginatedList<T> {
+    object: "list"
+    results: T[]
+    next_cursor: string | null
+    has_more: boolean
+}
+
+/**
+ * Copied from:
+ * https://github.com/makenotion/notion-sdk-js/blob/main/src/helpers.ts#L47
```

Comment:
Some anti-link-rot 😛 

```suggestion
 * https://github.com/makenotion/notion-sdk-js/blob/7950edc034d3007b0612b80d3f424baef89746d9/src/helpers.ts#L47
```

Reply:
Thanks!